docs/getting-started/quickstart/README.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Quickstart with Github Codespaces
+---
+
+# EventStoreDB Quickstart using GitHub Codespaces
+
+This quickstart will guide you through getting started with EventStoreDB using [GitHub Codespaces](https://github.com/features/codespaces).
+
+::: info
+GitHub Codespaces provides an instant and preconfigured development environment in your browser for this quickstart.
+:::
+
+## Objectives
+
+In this quickstart, you will:
+
+- Set up a Codespace development environment.
+- Start an EventStoreDB server using Docker.
+- Append an event to EventStoreDB with sample code.
+- View the appended event using the Admin UI.
+- Read the appended event with sample code using the EventStoreDB client.
+
+## Prerequisites
+
+Before starting, ensure you have the following:
+
+- A GitHub account to use GitHub Codespaces.
+- Basic knowledge of one of the development languages/platforms below.
+- Familiarity with command-line operations.
+
+## Step 1: Set up Your Codespace
+
+1. Choose one of development languages/plaforms below and click the Codespaces link:
+
+::: tabs#dev-language-platform
+
+@tab Select >
+
+
+@tab Python
+|Language/Platform|Link to Codespaces|
+|-----|--------|
+|<div style="display: flex; justify-content: center; align-items: center;"><img src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/python/python-original-wordmark.svg" alt="Python" style="height: 64px;" /></div>|[![Open in Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/EventStore/EventStoreDB-From-Scratch-Python?ref=stephen-getting-started&devcontainer_path=.devcontainer%2Fquickstart%2Fdevcontainer.json)|
```

Comment:
what's the `ref` used for in the URL ?

Reply:
The ref is used to point to the branch where the code is committed to. Since my branch in the "from scratch" repo (not the doc repo) isn't pushed to main yet, I am pointing this to my branch for now. As soon as my code is merged, I will point this back to main.

Reply:
@ylorph just fyi, i've removed `ref` so that it points to main now.